tests/ballerina-unit-test/src/test/resources/test-src/taintchecking/propagation/iterable-within-iterable.bal

```diff
@@ -0,0 +1,30 @@
+json globalVar;
+
+function main (string... args) {
+    jsonData.address_components
+    .filter((json comp) => boolean {return comp.types.filter((json compType) => boolean {secureFunction("untainted", "untainted"); return compType.toString() == "street_number";}).count() > 0; })
```

Comment:
Shall we reformat this line and method signatures in this file

Reply:
+1